example/mysite/docker_settings.py

```diff
@@ -1,28 +1,47 @@
+# An example settings file for deploying a Django app in a Docker container.
+# Uses environment variables to configure the majority of settings. This
+# pattern is sometimes attributed to the '12factor' app guidelines:
+# https://12factor.net/config
+
 from __future__ import absolute_import
 
 import environ
 
+# Import the existing settings file, we'll work from there...
 from .settings import ALLOWED_HOSTS, SECRET_KEY
 from .settings import *  # noqa
 
+# We use django-environ here to make working with environment variables a bit
+# easier: https://github.com/joke2k/django-environ
 env = environ.Env()
 
 SECRET_KEY = env.str('SECRET_KEY', default=SECRET_KEY)
 DEBUG = env.bool('DEBUG', default=False)
 ALLOWED_HOSTS = env.list('ALLOWED_HOSTS', default=ALLOWED_HOSTS)
 
 DATABASES = {
+    # Set up the database using a "database URL". See dj-database-url for more:
+    # https://github.com/kennethreitz/dj-database-url
```

Comment:
Here also it might help to clarify that this is a dependency that needs to be added.